README.md

```diff
@@ -1,2 +1,59 @@
-# portfolio-mobile-skeleton
-The project is to build a mobile website using the template chosen in the previous step.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio
+
+
+> The project is to build a mobile website using the template chosen in the previous step..
+
+
+## Built With
+- HTML
+- CSS
+
+## Live Demos
+
+[Live Demo Link]()
+
```

Comment:
This link is broken Would you kindly fix it? Your README should be written without any broken links so it gives a good first impression and maximizes the potential of your product to your users.